atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/queue/AbstractSweepQueueTest.java

```diff
@@ -165,14 +166,12 @@ void putTimestampIntoTransactionTable(long ts, long commitTs) {
 
     static CellReference getCellRefWithFixedShard(long seed, TableReference tableRef, int shards) {
         byte[] rowName = PtBytes.toBytes(seed);
-        int counter = 0;
-        CellReference cellReference;
-        do {
-            byte[] colName = PtBytes.toBytes(counter);
-            cellReference = CellReference.of(tableRef, Cell.create(rowName, colName));
-            counter++;
-        } while (IntMath.mod(cellReference.hashCode(), shards) != 0);
-        return cellReference;
+
+        return IntStream.iterate(0, i -> i + 1)
```

Comment:
Is there any concern here that this implementation becomes more expensive as the number of shards increases?

Reply:
Just realized this seems to only be used for tests